modules/social_features/social_event/social_event.module

```diff
@@ -1126,3 +1128,129 @@ function _social_event_views_update($view_config, $display) {
   $data['display'][$display]['display_options']['filters'] = $new_filters;
   $config->setData($data)->save();
 }
+
+/**
+ * Returns a description array for the field_enroll_method options.
+ *
+ * @return string
+ *   The render array containing the description.
+ */
+function social_event_enroll_method_description($key) {
+  $description = '';
+
+  // We need it to be specified otherwise we can't build the markup.
+  if (empty($key)) {
+    return $description;
+  }
+
+  // Add explanatory descriptive text.
+  switch ($key) {
+    case '1':
```

Comment:
would be cool to make a `const` out of this, so it's understandable for everybody what these keys mean.